### PR TITLE
Add Style to The awesome books App

### PR DESCRIPTION
**Pull request by BonkeGcobo**
Here we added some styling on the awesome book app

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,15 +11,16 @@
 
 <body>
```
Comment (SajjadAhmad14) [inline index.html:12]: Please add project descriptions of the project under the project title in README.

File: index.html [hunk 2]
```diff
@@ -11,15 +11,16 @@
...
    
 
   <section class="Books">
+    
   </section>
 
-
   <section class="AddingBook">
+    <h1>Add a New Book</h1>
     <form class="form">
```
Comment (SajjadAhmad14) [inline index.html:24]: Please follow the wireframe for the form to add books.  The button should be on the right side of the form and please give a bit more height and width to input fields.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Bonke, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by hiromataba — APPROVED** [review]
## Status: Approved :+1:  :tada: 
Hi @BonkeGcobo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Suggestions
Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better.

- [ ] [OPTIONAL] Kindly replace the picture in your README file with the screenshot of your project

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**